articles/moricho.re

```diff
@@ -0,0 +1,87 @@
+= Goによるコンテナランタイム自作入門
+
+== はじめに
+こんにちは、@_moricho_@<fn>{_moricho_}です。
+みなさんコンテナ使ってますか？最近ではDockerやKubernetesなど、業務の中でもすっかりコンテナ技術が浸透してきています。またAWSのLambdaなども実は裏でコンテナ技術が使われています。
```

Comment:
**[textlint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Disallow to use "？". (ja-technical-writing/no-exclamation-question-mark)